helpers/colour-contrast/accessibility.js

```diff
@@ -0,0 +1,19 @@
+import { contrastColors, contrastRGB } from './contrast.js';
+
+const contrastMin = 4.5;
```

Comment:
It might be nice to have a few variables here corresponding to the different WCAG A/AA/AAA requirements. I might suggest:

```
export const contrastNormalText = 4.5;
export const contrastLargeText = 3;
export const contrastNonText = 3;
```

Reply:
We don't typically aim for AAA, so I left those out.